### PR TITLE
Update codeql-analysis.yml to use .NET 6

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -36,13 +36,13 @@ jobs:
     #    and modify them (or add more) to build your code if your project
     #    uses a compiled language
 
-    - name: Setup .NET 5.0.408
-      uses: actions/setup-dotnet@v2.1.0
+    - name: Setup .NET 6.0.301
+      uses: actions/setup-dotnet@v1.7.2
```
Comment (jennyf19) [inline .github/workflows/codeql-analysis.yml:40]:
```suggestion
      uses: actions/setup-dotnet@v2.1.0
```

File: .github/workflows/codeql-analysis.yml [hunk 2]
```diff
@@ -36,13 +36,13 @@ jobs:
     #    and modify them (or add more) to build your code if your project
     #    uses a compiled language
 
-    - name: Setup .NET 5.0.408
+    - name: Setup .NET 6.0.301
       uses: actions/setup-dotnet@v2.1.0
       with:
-        dotnet-version: 5.0.408
+        dotnet-version: 6.0.301
```
Comment (jmprieur) [inline .github/workflows/codeql-analysis.yml:42]: can we use 6.x ?

**Review by jmprieur — APPROVED** [review]
LGTM
Thanks @jennyf19

File: .github/workflows/codeql-analysis.yml (hunk 2 above)
Comment (jennyf19) [reply .github/workflows/codeql-analysis.yml:42]: yes, but sometimes it's not happy with a wildcard

**Pull request merged**